### PR TITLE
Update Telescope overview and remove broken links

### LINKED ISSUE #3180: Update Telescope Overview docs and remove broken links
While working on translating the [Telescope Overview docs](https://github.com/Seneca-CDOT/telescope/blob/master/src/docs/docs/overview.md) I found some broken links:

- http://zenit.senecac.on.ca/~chris.tyler/planet/ redirects to https://matrix.senecacollege.ca/

- https://people.gnome.org/~jdub/bzr/planet/devel/trunk/ appears twice in the page and returns a 404.

> Our current Planet is dying

Judging by the broken links the old Planet has died!

Looks like the page was written in October 2019, so we can take this opportunity to update the English version of the page before continuing with all the translations (https://github.com/Seneca-CDOT/telescope/issues/2962).

### PR DESCRIPTION
**Pull request by tcvan0707**
## Issue This PR Addresses

Fixes #3180 

## Type of Change

- [ ] **Bugfix**: Change which fixes an issue
- [ ] **New Feature**: Change which adds functionality
- [x] **Documentation Update**: Change which improves documentation
- [ ] **UI**: Change which improves UI

## Description

This PR is going to update the Overview page of Telescope by removing broken links, and adding the alternative source for our Planet.

## Steps to test the PR

- Change your current directory to /src/docs: `cd src/docs`
- Run `pnpm start`
- Go to `localhost:4631` on your web browser

## Checklist

- [ ] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [x] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

File: src/docs/docs/overview.md [hunk 1]
```diff
@@ -32,14 +32,14 @@ various "feed" technologies (RSS, Atom, CDF) to allow blog posts from different
 platforms to be aggregated into a single page that was constantly updated with
 the latest posts by people within a particular community.
 
-[Written in Python by Jeff Waugh and Scott James Remnant](https://people.gnome.org/~jdub/bzr/planet/devel/trunk/),
+Written in Python by Jeff Waugh and Scott James Remnant,
```
Comment (aserputov) [inline src/docs/docs/overview.md:35]: Why did you remove the link ?

**Review by aserputov — APPROVED** [review]

File: src/docs/docs/overview.md (hunk 1 above)
Comment (tcvan0707) [reply src/docs/docs/overview.md:35]: It directs us to `404 Not Found Page`, and the content of this link I assume that it's already in [this](https://en.wikipedia.org/wiki/Planet_(software))

File: src/docs/docs/overview.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ something unique to "me."
 To better enable the discovery of blogs within our community, we set up an open
 source blog [Planet](<https://en.wikipedia.org/wiki/Planet_(software)>): an aggregated
 feed of blog posts from Seneca faculty and students working on open source in a
-single page. Our blog Planet currently lives at [http://zenit.senecac.on.ca/~chris.tyler/planet/](http://zenit.senecac.on.ca/~chris.tyler/planet/).
+single page. Our blog Planet currently lives at [https://telescope.cdot.systems/planet](https://telescope.cdot.systems/planet).
```
Comment (DukeManh) [inline src/docs/docs/overview.md:23]:
```suggestion
single page. Our blog Planet currently lives at [Planet CDOT](https://telescope.cdot.systems/planet).
```

**Review by TueeNguyen — APPROVED** [review]

**Review by JerryHue — APPROVED** [review]
I cannot speak for the Vietnamese page, aside from that, everything else is good!

**Comment by JerryHue** [comment]
@tcvan0707, I'll be rebasing your branch from GitHub and then I will merge it.
